### ISSUE TITLE
is this still being maintained?

### ISSUE THREAD

**Issue by ctfrancia** [rodgc/ngx-socket-io#49]
there are some things that I would like to do to improve including some issues but I see here that there are pull requests from 2018...

**Comment by Helveg** [comment]
@rodgc Could you let us know about the status of the project? Maybe give someone else access to the repository as well.

**Comment by sucotronic** [comment]
As @Helveg  mentioned, if @rodgc resources are low, we can contribute to maintain the project ;) For now I'm going to fork in order to be able to integrate in an angular 9 project.

**Comment by Helveg** [comment]
I'll be watching your fork then suco. I'm also interested in the Angular 9 aspect.

**Comment by sucotronic** [comment]
@Helveg I just did a PR with one line fix to be angular 9 compatible :P

**Comment by MarcosEich** [comment]
I'm also interested in this project. Is it still maintained?

**Comment by Helveg** [comment]
No this project is no longerr maintained, we could maintain it on  @sucotronic's fork?

**Comment by MarcosEich** [comment]
I think so, should we setup a new npm package?

**Comment by Helveg** [comment]
From https://www.npmjs.com/policies/disputes:

> tl;dr
Get the author email with npm owner ls <pkgname>
Email the author, CC support@npmjs.com
After 4 weeks, if there's no resolution, we'll address it.
Don't squat on package names, user names or organization names. Publish code or move out of the way.

>When to use this process
This process is an excellent way to:

>Adopt an "abandoned" package
Adopt a package created from your project, published by someone else
Report "squatting"
Report a deliberately misleading or confusing package name

I think that would be better for the exposure of the package.

**Comment by Helveg** [comment]
@ctfrancia, @sucotronic I think this makes 4 people interested in maintaining this project and sucotronic already has a fork. Do we all agree that we propose to adopt the abandoned package on npm and continue the project on @sucotronic's fork if we in this 4 week period hear nothing from @rodgc?

Who's going to send the mail to NPM?

**Comment by martinpotel** [comment]
Hi all, I'm also interested in this project. So, it would be great to maintain it on @sucotronic's fork and I'm willing to help you guys with that.

**Comment by MarcosEich** [comment]
@Helveg let's do it. Let's use @sucotronic fork and email npm and assume the project if @rodgc is not interested in maintaining it anymore.

**Comment by Helveg** [comment]
I went ahead and sent the email to @rodgc with npm's support in CC

**Comment by rodgc** [comment]
Hi All, Sorry for not responding all your messages ASAP, I have a lot of work and can't find a time to continue maintain it as I would like.

If you guys want to help me to maintain it i would appreciate it a lot

**Comment by MarcosEich** [comment]
Sounds good to me, I would be happy to participate.

**Comment by ctfrancia** [comment]
I would be happy to contribute to the project, I was just unaware of the status as there are pull requests waiting and issues are continuing to pile. I don't think that anyone doubts you have a lot of work to do @rodgc

**Comment by johaven** [comment]
Any updates ?

**Comment by ctfrancia** [comment]
there are some pull requests, we just need to wait for the owner to merge.

**Comment by johaven** [comment]
You first post is dated February ... It would be necessary to delegate the deposit ...

**Comment by Helveg** [comment]
Yes the update is that Rod has replied and given push access at least to me (but I currently don't have any time either) and possibly others? And that these people will help maintain this repository.

**Comment by dhaeb** [comment]
@All Would be great if you note that on the main projects page that the project is currently not maintained or is currently NOT working with angular 9 - acutally I tried it and got only weired 426 / ssl errors which I thought were my fault. So I hope that others don't waste so much time like me. 
Even better, it would be great if you revive the project - I liked it's API und close relation to socket.io. 
Thank you!

**Comment by ctfrancia** [comment]
@dhaeb you can pull the fork from sucotonic as he has resolved it, and submitted a PR however the fix hasn't been merged yet. So if you need it right now then I suggest cloning from sucotronic

**Comment by sucotronic** [comment]
I've merged my own PR, don't feel very ok with that, but it'll work for now.

**Comment by ctfrancia** [comment]
At some point, I don't know when, I'll add a CI/CD and tests to help with integrity, but, I don't know when. Hopefully when I have some spare time.

**Comment by dhaeb** [comment]
@ctfrancia @sucotronic 
Thank you! 
Do you plan to update the NPM dependencies? 
https://www.npmjs.com/package/ngx-socket-io
I'm going than with building it locally and trying it out. If I encounter any problems, I will open an seperate issue!
Best of luck with this project, hope there will be some dev joining and maintaining it!
